### ISSUE TITLE
Field maxcommrate is not mutated

### ISSUE THREAD

**Issue by anton-trunov** [Zilliqa/staking-contract#100]
Commit b710edc8b0676e0ea40dc5422d7d2f8c8e1552b4

Is there a missing transition that should set the `maxcommrate` field?
If this is not the case then it should perhaps be made a constant or a contract immutable parameter.

**Comment by bb111189** [comment]
Yes, I think it can be made constant. The purpose of `maxcommrate` is to ensure that operators do not set commission rate is that more than 100%, resulting in potential unexpected behavior
